src/main/java/mods/eln/Eln.java

```diff
@@ -4642,6 +4756,238 @@ private void registerRawCable(int id) {
         }
     }
 
+    private void registerArc(int id) {
```

Comment:
The arc furnace should be a different PR. Or at least needs to be finished before I'll accept this one.